xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.cpp

```diff
@@ -41,7 +34,14 @@ bool CDVDInputStream::Open(const char* strFile, const std::string &content, bool
   // but keep them in m_url.
   url.SetProtocolOptions("");
   m_strFileName = url.Get();
+}
 
+CDVDInputStream::~CDVDInputStream()
+{
+}
+
+bool CDVDInputStream::Open(const std::string &content, bool contentLookup)
```

Comment:
The only issue with using CFileItem is some clients don't currently have a CFileItem, but I guess we can create a CFileItem( path, false) on stack for those.